pgmpy/base/DirectedGraph.py

```diff
@@ -177,12 +171,11 @@ def moralize(self):
 
         Examples
         --------
-        >>> from pgmpy.base import DirectedGraph
         >>> G = DirectedGraph([('diff', 'grade'), ('intel', 'grade')])
         >>> moral_graph = G.moralize()
-        >>> moral_graph.edges()
+        >>> moral_graph.edges() # doctest: +IGNORE_RESULT
```

Comment:
@yashu-seth Actually sorting is required because output may not be same each time. But we can use doctests with unittests to overcome this.( as mentioned in below comment)